File: 3Arrays/Array_operation code/Traversal/Traversal.c
#include <stdio.h>

int main(){
    int n;
    printf("Enter the size of the array");
    scanf("%d",&n);
    int arr[n];
   
    for(int i=0;i<n;i++){
        printf("Enter the elements :");
        scanf("%d",&arr[i]);
    }
    printf("The elements of the Array are:");
    for(int i=0;i<n;i++){
        printf("%d",arr[i]);
    }

  return 0;
}

File: 3Arrays/Array_operation code/Traversal/Traversal.java
package Traversal;
import java.util.*;
public class Traversal{
    public void display(int arr[],int n){
        for(int i=0;i<n;i++){
            System.out.print(arr[i]+" ");
        }
        System.out.println();
    }
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter the lengtyh of the ArrayList: ");
        int n=sc.nextInt();
        int [] arr=new int[n];
        for (int i=0;i<n;i++){
            System.out.println("Enter the element:");
            arr[i]=sc.nextInt();
        }
        // System.out.println("The elements are:");
        // for (int a: arr){
        //     System.out.print(a);
        //     System.out.print("");
        // }

        Traversal t=new Traversal();
        t.display(arr,n);
    }
}